client/src/assets/icons/JavaScript-icon.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { Fragment } from 'react';
 import { useTranslation } from 'react-i18next';
 
-function JavaScriptIcon(props) {
+function JavaScriptIcon(props: any & React.SVGProps<SVGSVGElement>) {
```

Comment:
```suggestion
function JavaScriptIcon(props: React.SVGProps<SVGSVGElement>) {
```
If you use `any & AnotherType` it just collapses to the `any` type. Basically, `any` means "don't type check this", so asking the type to have all the properties of `AnotherType` doesn't really do anything.